client/my-sites/invites/invite-accept-logged-out/index.jsx

```diff
@@ -174,11 +177,13 @@ class InviteAcceptLoggedOut extends Component {
 				loginUser: this.loginUser,
 			} );
 		}
-
+		const userInvitePasswordless = this.props.userInvitePasswordless;
 		return (
 			<div>
 				<SignupForm
-					redirectToAfterLoginUrl={ window.location.href }
+					displayNameInput={ ! userInvitePasswordless }
+					displayUsernameInput={ ! userInvitePasswordless }
+					displayPasswordInput={ ! userInvitePasswordless }
```

Comment:
We should use the `isPasswordless` property

Reply:
AFAIK, The [Passwordless](https://github.com/Automattic/wp-calypso/blob/trunk/client/blocks/signup-form/index.jsx#L1223) component is used on onboarding flows. They handle user creation and are coupled to a specific flow, this is why we have this prop.

Since we need user creation and accept the invite. I am passing these new props down just to hide and prevent validation on Username, Name, and Password fields and keep the component working as before and make It look like a passwordless form.